deploy/materialized_view_identifier.sql

```diff
@@ -6,51 +6,37 @@ begin;
 create materialized view ggircs_swrs.identifier as (
   with import_xml as (
     select _ghgr_import.xml_file as source_xml,
-           _ghgr_import.id         as ghgr_id,
-           _ghgr_import.imported_at
+           _ghgr_import.id         as ghgr_import_id,
     from ggircs_swrs.ghgr_import as _ghgr_import
     order by _ghgr_import.id asc
   )
-  select row_number() over (order by ghgr_id asc) as id,
-         ghgr_id,
-         report_details.swrs_report_id,
-         report_details.swrs_facility_id,
-         identifier_type,
-         identifier_value,
-         -- imported_at,
+  select row_number() over (order by ghgr_import_id asc) as id,
+         ghgr_import_id,
+         swrs_identifier.*,
          row_number() over (
            partition by swrs_facility_id, identifier_type
            order by
-             ghgr_id desc,
+             ghgr_import_id desc,
              identifier_value desc
            ) as swrs_identifier_history_id
   from import_xml,
        xmltable(
-           '/ReportData'
+           '//Identifier'
            passing import_xml.source_xml
            columns
-           swrs_report_id numeric(1000,0) path '//descendant-or-self::ReportID[normalize-space(.)]' NOT NULL,
-           swrs_facility_id numeric(1000,0) path '//descendant-or-self::FacilityId[normalize-space(.)]' NOT NULL
-        ) as report_details,
-       xmltable(
-           '/ReportData//descendant-or-self::Identifier'
-           passing import_xml.source_xml
-           columns
-             -- _report.swrs_report_id,
-             -- _report.swrs_facility_id,
-             identifier_type varchar(1000) path '//descendant-or-self::IdentifierType[normalize-space(.)]' NOT NULL,
-             identifier_value varchar(1000) path '//descendant-or-self::IdentifierValue[normalize-space(.)]' NOT NULL
-         )
+             swrs_facility_id numeric(1000,0) not null path '//FacilityId[normalize-space(.)]',
```

Comment:
We assume the FacilityId appears exactly once so we use the root descendant axis to parse it from the xml

Reply:
we don't need to shimmy up the tree with ancestor-or-self?

Reply:
We're making the assumption it appears exactly once per document so yes we might have to if that turns out not to be true. The `xmltable` extractor will error if it gets two values, and the not null declaration will error if it gets no values so we'll know soon enough.